terraform/eks-cluster/commands/create.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -e
+
+# Usage
+# ./command/create.sh [--all|--cluster|--app|--run] [--values /path/to/custom-values.yaml] [--helm /path/to/local/helm/chart]
+#
+# Note: Must be executed from the /terraform/eks-cluster directory.
+#
+# Examples:
+#
+# Create cluster, app and run bazel
+# ./command/create.sh --all
+#
+# Create just the cluster
+# ./command/create.sh --cluster
+#
+# Create the app and run bazel
+# ./command/create.sh --app --run
+#
+# Create cluster, app with custom values.yaml and local helm chart, and run bazel
+# ./commands/create.sh --all --values ../buildbuddy-enterprise/values/demo.yaml --helm ~/Code/buildbuddy-helm/charts/buildbuddy-enterprise/
+
+# Defaults
+VALUES_PATH=../buildbuddy-enterprise/values/rbe-minimal.yaml
+HELM_CHART=buildbuddy/buildbuddy-enterprise
+
+# Check if there are arguments
+if [ $# -eq 0 ]; then
+    echo "Missing arguments, try --all, --cluster, --app, --run, or some combination of those"
+fi
+
+# Parse arguments
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        -v|--values) VALUES_PATH="$2"; shift ;; # -v or --values, optional - path to a custom helm values.yaml file
+        -h|--helm) HELM_CHART="$2"; shift ;; # -h or --helm, optional - a custom helm chart to use, can be a directory
+        -c|--cluster) CREATE_CLUSTER=1 ;; # -c or --cluster, optional - create an eks kubernetes cluster
+        -a|--app) CREATE_APP=1 ;; # -a or --app, optional - deploy the buildbuddy app
+        -r|--run) SHOULD_RUN=1 ;; # -r or --run, optional - run a test bazel build against the cluster
+        --all) ALL=1 ;; # -all, optional - create cluster, deploy app, and run test bazel build
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+done
+
+# Check if we should create a cluster
+if [ $CREATE_CLUSTER ] || [ $ALL ]; then
```

Comment:
`$CREATE_CLUSTER == 1` might be needed here, at least on my version of bash:

```bash
$ foo=0; if [ $foo ]; then echo ZOINKS; fi
ZOINKS
```

Reply:
I hate that these things aren't standard - should be fixed